myapp/app/controllers/task_controller.rb

```diff
@@ -0,0 +1,55 @@
+class TaskController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def new
+    @task = Task.new
+    @labels = Label.all
+  end
+
+  def create
+    @task = Task.new(task_params)
+    @task.user_id = 1
```

Comment:
User function will be implemented later step, so you specified user_id?

Reply:
Previous step-7 had "user_id" column as NOT NULL. so i needed to provide value.
so do you want me to migrate to NULL or removing user_id column?

Reply:
I just wanted to confirm, it is ok